app/models/case_log.rb

```diff
@@ -35,6 +35,7 @@ class CaseLog < ApplicationRecord
 
   scope :for_organisation, ->(org) { where(owning_organisation: org).or(where(managing_organisation: org)) }
   scope :filter_by_status, ->(status) { where status: status }
+  scope :filter_by_year, ->(year) { where(startdate: Time.zone.local(year, 4, 1)...Time.zone.local(year.to_i + 1, 4, 1)) }
```

Comment:
Think you need the `year.to_i` on the start date selection as well since I'm assuming it comes in as a string also? Should that be causing a spec to fail?

Reply:
Also should the end date be `Time.zone.local(year.to_i + 1, 3, 31).end_of_day`?